### DISCUSSION TITLE
[Help] 请问这是部署在vps上还是部署在本地虚拟机里呢？

### DISCUSSION THREAD

**Discussion by himonster1** [kkkgo/PaoPaoDNS#39] [问答和帮助]
### 在提交之前，请确认

- [X] 我已经尝试搜索过Issue和看过参数说明，但没有找到相关问题。
- [X] 我正在使用最新的docker镜像版本（可以尝试`docker pull sliamb/paopaodns:latest`后重新创建容器，如果不确定的话可以尝试使用`sliamb/paopaodns:dev`）。

### 脚本自检日志

```txt
部署在vps还是部署在内网呢？
```

### 问题描述和复现步骤

部署在vps还是部署在内网呢？

**Comment by kkkgo** [comment]
取决于你的需求，大部分场景下适合部署在本地内网。如果你想搭建一个对外的公共DNS，则适合部署在VPS。
